pkgs/by-name/qs/qsp/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  oniguruma,
+}:
+stdenv.mkDerivation {
+  name = "qsp";
+  version = "0-unstable-2024-11-27";
+  src = fetchFromGitHub {
+    owner = "QSPFoundation";
+    repo = "qsp";
+    rev = "f6ede7f8756e49604de056fcbdfe99fa4abd4812";
+    sha256 = "MoNam2IFnLpk02tKp+lkl4l+mBiaWNPhFc3/n4zUHcw=";
+  };
+  buildInputs = [ oniguruma ];
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+  cmakeFlags = [ (lib.cmakeBool "USE_INSTALLED_ONIGURUMA" true) ];
+  meta = with lib; {
```

Comment:
Still need to get rid of with lib; here

Reply:
Oh, I saw other packages doing that, so I assumed it was standard. Fixed